spyder/plugins/ipythonconsole/plugin.py

```diff
@@ -678,8 +668,7 @@ def exec_files(
             current_client,
             clear_variables,
             console_namespace,
-            focus_to_editor=True,
-            method='debugfile' if debug else 'runfile',
+            method=run_method,
             force_wdir=force_wdir
```

Comment:
Could you check if the `force_wdir` argument here is correct as it's defined above? I found out that you added it in PR https://github.com/spyder-ide/spyder/pull/19181.